spring-integration-file/src/main/java/org/springframework/integration/file/dsl/FileTransferringMessageHandlerSpec.java

```diff
@@ -34,6 +29,11 @@
 import org.springframework.messaging.Message;
 import org.springframework.util.Assert;
 
+import java.nio.charset.Charset;
+import java.util.Collections;
+import java.util.Map;
+import java.util.function.Function;
```

Comment:
You can't reorder imports. That's where Checkstyle fails for you.
Please, consider do not do auto-formatting in your IDE before pushing commits.

Reply:
fixed this issue.